form.yml

```diff
@@ -155,17 +382,37 @@ cluster:
             $name: Parameter value
             $optional: true
 
-  add_exporter:
-    $name: Add HA cluster metrics exporter
+  remove_checkbox:
+    $name: Remove cluster node(s)
     $type: boolean
+    $optional: true
     $default: false
-    $help: Mark if you want add the HA cluster metrics exporter
-  ha_exporter:
-    $name: HA cluster metrics exporter
+  remove:
+    $name: Remove node(s) from cluster
+    $visibleIf: .remove_checkbox == true
     $optional: true
-    $visibleIf: .add_exporter == true
-    $type: group
-    exposition_port:
-      $name: HA cluster exporter (hawk-apiserver) exposition port
-      $type: text
+    $type: edit-group
+    $itemName: ""
+    $minItems: 0
+    $prototype:
+      $name: Cluster node to be removed
       $optional: true
+      $type: text
+
+  monitoring_enabled:
```

Comment:
I would add here the `optional` entry. This way, if the value is set to false, won't appear in the pillar file

Reply:
Forget about this. Just removing the initial conditions in the `pre_validation` about the monitoring should work fine